### PR TITLE
method check for remote videos

### PR DESCRIPTION
**Pull request by ajstanley**
# What does this Pull Request do?

Currently if we add a remote video media type Islandora whitescreens when it tries to use the non-existent referencedEntities() method.  This small change just checks for the method's existence before attempting to run.

# What's new?
One new check before execution.

# How should this be tested?
Add a remote video as a media. (You'll have to add mediaOf and mimeType as existing fields to the mediaType).

Try to display that media and you'll see a whitescreen.
Apply this pull, and you should see your embedded video.

# Interested parties
@Islandora-CLAW/committers

### CODE REVIEW

File: src/EventSubscriber/MediaLinkHeaderSubscriber.php [hunk 1]
```diff
@@ -71,7 +71,8 @@ protected function generateMediaLinks(MediaInterface $media) {
     $source_field = $type_configuration['source_field'];
 
     if (empty($source_field) ||
-        !$media->hasField($source_field)
+        !$media->hasField($source_field ||
```
Comment (dannylamb) [inline src/EventSubscriber/MediaLinkHeaderSubscriber.php]: you're missing a closing parenthesis here before your `||`

Comment (ajstanley) [reply src/EventSubscriber/MediaLinkHeaderSubscriber.php]: It's three different conditions with logical ORs stuck between them,
if
  empty($source_field)
or
  !$media->hasField($source_field 
or 
  !method_exists($media->get($source_field), 'referencedEntities')

Comment (ajstanley) [reply src/EventSubscriber/MediaLinkHeaderSubscriber.php]: d'oh

**Comment by dannylamb** [comment]
Just waiting on Travis...

**Pull request merged**